engine/request-handling-utils.js

```diff
@@ -0,0 +1,30 @@
+const request = require('request')
```

Comment:
Where does this code come from? Is it supposed to be moved from somewhere? If yes, the deletion from there should be in this commit as well.